mmv1/compiler.rb

```diff
@@ -88,6 +90,9 @@
   opt.on('-g', '--no-docs', 'Do not generate documentation') do
     generate_docs = false
   end
+  opt.on('--openapi-generate', 'Generate MMv1 YAML from openapi directory') do
```

Comment:
nit: Could we be internally consistent about `openapi` vs `open_api`? https://swagger.io/specification/ uses "openapi" for the version number, so I'd lean towards that one personally (but if there's a better precedent to follow, we should use that). 

Mostly `open_api_parser` should be `openapi_parser` if so.

Reply:
Renamed openapi_generate to not conflict with the gem openapi_parser that I'm using to read the actual specs